components/contracts/contractRouting.ts

```diff
@@ -324,6 +324,17 @@ contractRoutingRouter.post(
     },
 )
 
+contractRoutingRouter.post(
+    "/ReportContract",
+    jsonMiddleware(),
+    (
+        req: RequestWithJwt<never, { contractId: string; reason: number }>,
+        res,
+    ) => {
+        res.json({})
+    },
+)
```

Comment:
This will also attach /ReportContract to /ContractsService

Reply:
Added a separate router for reports.

Reply:
Shouldn't this be removed then?

Reply:
I forgor 💀 Removed now.